recipe_Menager.py

```diff
@@ -2,26 +2,105 @@ class RecipeManager:
     def __init__(self):
         self.recipes = []
 
-    @property
     def display_recipes(self):
         for recipe in self.recipes:
-            print(recipe)
-            return self.recipes
-            
-    @display_recipes.getter
-    def display_recipes(self):
-      return str(self.recipes)
+            print(recipe._title) #title instead the name 
+            return 
+        if not self.recipes:
+            print("No recipes to display.")
+            return
+
+
+    def add_recipe(self, recipe_to_add: Recipe):
+        self.recipes.append(recipe_to_add)
+    def add_recipe_from_input(self):
+        title = input("Enter the title of the dish: ")
+        description = input("Enter the description: ")
+        no_servings = int(input("Enter the number of servings: "))
+        calories_per_portion = int(input("Enter the calories per portion: "))
+        ingredients = input("Enter the ingredients: ").split(",")
+        instructions = input("Enter the instructions: ").split(",")
+
+        new_recipe = Recipe(title)
+        new_recipe._description = description  # Set description directly
+        new_recipe.set_no_servings(no_servings)
+        new_recipe.set_calories_per_portion(calories_per_portion)
+        for ingredient in ingredients:
+            new_recipe.set_ingredients(ingredient.strip())
+        for instruction in instructions:
+            new_recipe.set_instructions(instruction.strip())
+        
+        self.add_recipe(new_recipe)
+        print("Recipe added.")
+        
+
+    def update_recipe(self, recipe_to_update: Recipe, updated_recipe: Recipe):
+        recipe_to_update.set_description(updated_recipe.get_description())
+        recipe_to_update.set_no_servings(updated_recipe.get_no_servings())
+        recipe_to_update.set_calories_per_portion(updated_recipe.get_calories_per_portion())
+        recipe_to_update.set_ingredients(updated_recipe.get_ingredients())
+        recipe_to_update.set_instructions(updated_recipe.get_instructions())
+    def update_recipe_from_input(self):
```

Comment:
Again line space required. Again this is something for debug/development/testing and should actually be here.